src/common/helpers.js

```diff
@@ -2,3 +2,17 @@ export const kelvinToCelsius = kelvin => {
   const celsius = kelvin - 273;
   return Number(celsius.toFixed(1));
 };
+
+export const groupQueryString = queryString => {
```

Comment:
Could be refactored to:
```
export const groupQueryString = queryString => 
    queryString
        .slice(1, queryString.length)
        .split('&')
        .reduce((acc, query) => {
            const [key, value] = query.split('=');
            acc[key] = value;
            return acc;
          }, {})
```

You don't have to explain each line, especially where code is self explanatory. Better provide a comment at the top, before defining your function with examples of input and ouput of your functon. See how it is done for docs autogenerators such as JSdoc. 


Later we will discuss other means to keep data (an alternative to passing it via query strings). You amy also need this data in other parts of the app, not only on Home page